frontend/src/components/Buttons/FilledButton/index.tsx

```diff
@@ -31,6 +32,7 @@ export const FilledButton: React.FC<ButtonProps> = ({
                 bgActive(colorPalette),
                 className
             )}
+            type={type ? type : 'button'}
```

Comment:
wouldn't 
```typescript
type ?? 'button'
```
be more idiomatic typescript?

Reply:
👍